usrp3/tools/scripts/uhd_image_builder_gui.py

```diff
@@ -542,6 +538,27 @@ def grc_populate_list(self, parent, files):
             self.show_not_xml_warning()
             return
 
+    def refresh_oot_dirs(self):
```

Comment:
@ejk43  With this function you are getting the OOT blocks from already generated Makefile.oot.inc, right? By removing the "open file" (see my last comment), isn't this requiring that the uhd_image_builder.py has to be run *before* the uhd_image_builder_gui.py for the OOT blocks to be picked up?